File: ruby/fizzbuzz.rb
def fizzbuzz(input_number)
	if input_number % 5 == 0 && input_number % 3 == 0
		puts "FizzBuzz"
	elsif input_number % 5 == 0
		puts "Fizz"
	elsif input_number % 3 == 0
		puts "Buzz"
	else
		puts input_number
	end
end

while true do
	print "Enter a value: "
	input_value = gets.chomp
	fizzbuzz(input_value.to_i)
end

File: go/fizzbuzz.go
package main

import "fmt"

func fizzbuzz(input int32) {
	if(input % 5 == 0 && input % 3 == 0) {
		fmt.Println("FizzBuzz")
	} else if(input % 5 == 0) {
		fmt.Println("Fizz")
	} else if(input % 3 == 0) {
		fmt.Println("Buzz")
	} else {
		fmt.Println(input)
	}
}

func main() {
	var input int32;
	for 1 == 1 {
		fmt.Print("Enter a number: ")
		fmt.Scanf("%d", &input)
		fizzbuzz(input)
	}
}
